### PR TITLE
fix(plugin-ris) fix parse function

### LINKED ISSUE #66: Parsing RIS does not handle leading whitespace (like U+FEFF)
## Problem

The RIS input parsing function fails if the first line (or any other line) starts with whitespace like U+FEFF.

## Solution

Trim whitespace of input RIS lines.

https://github.com/citation-js/citation-js/blob/ec3caf160ab87d0ec42bf33fe0119058c957737c/packages/plugin-ris/src/ris.js#L51-L52

### PR DESCRIPTION
**Pull request by soulchainer**
Closes #66 

I tried to run `yarn test` of this project, but in fails big time... I think this could be related with this being a multirepo and all that stuff. Any insight (just because I'm curious) about this?
I even used `nvm use v6`, for using the proper version of node this project states it supports, just in case it was because of my `node` version.

### CODE REVIEW

**Comment by soulchainer** [comment]
IDK what I have to do to improve the coverage here, if anything.
I understand that it's related to tests and that codecov (first time dealing with it) isn't happy because there is a decrease of coverage percentage, but I checked the tests for `plugin-ris` and I don't get them... plus what I commented previously about tests doesn't even run OK in my machine, but dependencies problems. In this project there are several package.json and it seems that only installing dependencies of the `plugin-ris` `package.json` is not enought, neither also installing dependencies of `citation-js`, so I'm clueless, right now >_<.

**Comment by larsgw** [comment]
Thank you!

To run the tests, you probably need to run `npx lerna bootstrap`. That installs (and links) the dependencies of the individual packages, while `npm install`, in this repository anyway, only installs generic devDependencies. After, `yarn test` and `npm test` should work. It's all in the new [`CONTRIBUTING.md`](https://github.com/citation-js/citation-js/blob/v0.5/CONTRIBUTING.md), but the one in the [`v0.5` branch](https://github.com/citation-js/citation-js/tree/v0.5), not the one in `master`. That's my bad, sorry for the confusion.

Either way, you don't have to worry about the code coverage.

File: packages/plugin-ris/src/ris.js [hunk 1]
```diff
@@ -48,9 +48,12 @@ export function parse (text) {
   let lastEntry
   let lastTag
 
-  for (let line of text.split('\n')) {
+  for (let line of text.split(/\r?\n/)) {
     line = line.trim()
-    if (!LINE_MATCH.test(line)) { lastEntry[lastTag] += line }
+    if (!LINE_MATCH.test(line)) {
+      if (lastEntry && lastTag) { lastEntry[lastTag] += ' ' + line.trim() }
```
Comment (soulchainer) [inline packages/plugin-ris/src/ris.js]: Hi, @larsgw. One question: why it's needed to `trim()` line here when `line`  already comes trimed from line `52`?

Comment (larsgw) [reply packages/plugin-ris/src/ris.js]: The second one can be removed, the line needs to be trimmed for `LINE_MATCH` mainly.

**Pull request merged**